enterprise_subsidy/apps/subsidy/models.py

```diff
@@ -165,7 +177,7 @@ def enrollment_client(self):
     def price_for_content(self, content_key):
         return self.catalog_client.get_content_metadata(content_key).get('price') * CENTS_PER_DOLLAR
 
-    def is_redeemable(self, learner_id, content_key, redemption_datetime=None):
+    def is_redeemable(self, content_key, redemption_datetime=None):
         return self.current_balance() >= self.price_for_content(content_key)
 
     def has_redeemed(self, learner_id, content_key, **kwargs):
```

Comment:
@iloveagent57 I was tending to my TODO list and one of the items I have from you is to rename "has redeemed" to something like "does redemption exist".  However, we're returning a "redemption" which is really a transaction, and bubbling it up in callers.  I think a better name would be "get_redemption" which doesn't imply that a boolean is returned.  thoughts?

Reply:
made the change to `get_redemption` as I described.